doc/example/prior_definition.ipynb

```diff
@@ -0,0 +1,243 @@
+    "\n",
+    "\\begin{align}\n",
+    "f(x, \\theta) = \\sum_{i=1}^{N} \\underbrace{100 \\cdot(x_{i}-x_{i-1}^2)^2}_{\\text{\"negative log-likelihood\"}} + \\underbrace{(x_{i-1}-1)^2}_{\\text{\"Gaussian log prior\"}}\n",
+    "\\end{align}\n",
+    "\n",
+    "as an example. Here we interpret the first term as the _negative log-likelihood_ and the second term as Gaussian _log-prior_ with mean $1$ and standard deviation $1/\\sqrt{2}$. \n",
+    "\n",
+    "Note that the second term is only equivalent to a the negative log-distribution of a Gaussian up to a constant."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Define the negative log-likelihood"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 26,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "n_x = 5\n",
+    "\n",
+    "\n",
+    "def rosenbrock_part_1(x):\n",
+    "    \"\"\"\n",
+    "    Implements obj. fct + gradient of the \"likelihood\" part.\n",
+    "    \"\"\"\n",
+    "    obj = sum(100.0*(x[1:] - x[:-1]**2.0)**2.0)\n",
+    "    \n",
+    "    grad = np.zeros_like(x)\n",
+    "    grad[:-1] += -400 * (x[1:] - x[:-1]**2.0) * x[:-1]\n",
+    "    grad[1:] += 200 * (x[1:] - x[:-1]**2.0)\n",
+    "    \n",
+    "    return (obj, grad)\n",
+    "\n",
+    "neg_log_likelihood = pypesto.Objective(fun=rosenbrock_part_1, grad=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Define the log-prior\n",
+    "\n",
+    "A prior on an individual paramater is defined in a `prior_dict`, which contains the following key-value pairs:\n",
+    "\n",
+    "* `index`: Index of the parameter\n",
+    "* `density_fun`: (Log-)posterior. (Scalar function!)\n",
+    "* `density_dx`: d/dx (Log-)posterior (optional)\n",
+    "* `density_ddx`: d^2/dx^2 (Log-)posterior (optional)\n",
+    "\n",
+    "A `prior_dicts` can be either obtained by `get_parameter_prior_dict` for several common priors, or defined by the user."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 27,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pypesto.objective.priors import get_parameter_prior_dict\n",
+    "\n",
+    "# create a list of prior dicts...\n",
+    "prior_list = []\n",
+    "mean = 1\n",
+    "std_dev = 1/np.sqrt(2)\n",
+    "\n",
+    "for i in range(n_x-1):\n",
+    "    prior_list.append(get_parameter_prior_dict(i, 'normal', [mean, std_dev]))\n",
+    "    \n",
+    "# create the prior\n",
+    "neg_log_prior = pypesto.objective.NegLogParameterPriors(prior_list)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Define the negative log-posterior and the problem\n",
+    "\n",
+    "The negative log-posterior is defined as an `AggregatedObjective`. Since optimization/visulazation is not the main focus of this notebook, the readers is referred to other examples for a more in depth presentation of these."
```

Comment:
```suggestion
    "The negative log-posterior is defined as an `AggregatedObjective`. Since optimization/visualization is not the main focus of this notebook, the reader is referred to other examples for a more in-depth presentation of these."
```